packages/components/src/components/icon/icon.tsx

```diff
@@ -89,6 +89,15 @@ export class Icon {
     );
 
     // TODO: wouldn't it be better to set alt attribute instead of aria-label?
-    return <img src={buildIconUrl(this.source || this.name)} {...parseAndGetAriaAttributes(this.aria)} alt="" />;
+    return (
+      <img
+        src={buildIconUrl(this.source || this.name)}
+        {...parseAndGetAriaAttributes(this.aria)}
```

Comment:
Instead of using aria-label, wouldn't the `alt` attribute good enough or even better?